core/src/main/scala/precog/AutoBump.scala

```diff
@@ -389,9 +389,14 @@ class AutoBump(
       pullRequest <- getOrCreatePullRequest(updateBranch)
       labels <- getLabels(pullRequest.number).compile.toList
       prChangeLabels = labels.flatMap(label => ChangeLabel(label.name)).toSet
-      highestChange = updateBranch.label.map(prChangeLabels + _).getOrElse(prChangeLabels).max
-      lowerChanges = (prChangeLabels - highestChange).toList
-      _ <- if (prChangeLabels.contains(highestChange)) Sync[F].unit else assignLabel(highestChange.label, pullRequest)
+      maybeHighestChange = updateBranch.label.map(prChangeLabels + _).getOrElse(prChangeLabels).toList.maximumOption
+      lowerChanges = maybeHighestChange.map(prChangeLabels - _).getOrElse(Set.empty).toList
+      _ <- maybeHighestChange match {
+        case Some(highestChange) if !prChangeLabels.contains(highestChange) =>
```

Comment:
The condition here isn't tested. It would require some changes on `TestEnv` that are possibly more complex than the rest of the PR, and I'm not sure `assignLabel` would fail if it adds an existing label anyway. So I'd prefer to have an existing error I can write a test case for.